src/framework/components/system.js

```diff
@@ -103,6 +111,9 @@ Object.assign(pc, function () {
                 case 'update':
                     this._erase(this._update, func, scope);
                     break;
+                case 'animationUpdate':
+                    this._erase(this._animationUpdate, func, scope);
+                    break;
```

Comment:
Just wondering why `pc.AnimComponent` needs a specific update event when `pc.AnimationComponent` didn't (I'm sure there's a good reason!) 😄

Reply:
It was mentioned that this execution order will need to be guaranteed for animation culling / IK.

Reply:
OK, fair enough.